src/NUnitTestAdapter/NavigationData.cs

```diff
@@ -0,0 +1,31 @@
+﻿// ****************************************************************
+// Copyright (c) 2016 NUnit Software. All rights reserved.
+// ****************************************************************
```

Comment:
I prefer this copyright header. Should we start using it elsewhere? I don't see the point in reproducing the license everywhere.

On an aside, thanks for this PR. The .NET Core runner needs to provide line numbers too. The .NET Core runner  has a class that does what Mono.Cecil does, but I like this architecture, so I am reusing a few of your classes and names.